__tests__/app.test.js

```diff
@@ -38,29 +38,74 @@ describe('GET Requests', () => {
             });
         });
     });
-    describe('GET /api/articles', () => {
+    describe.only('GET /api/articles', () => {
         test('/api/articles 200: responds with all articles & associated comment counts', () => {
-        return request(app)
-        .get('/api/articles')
-        .expect(200)
-        .then((res) => {
-            //Check Correct Length
-            expect(res.body.articles.length).toBeGreaterThanOrEqual(13);
-            //Check Sort Order
-            expect(res.body.articles).toBeSorted('created_at', 'desc');
-            expect(res.body.articles[12].title).toBe("Z");
-            //Check each element has correct props
-            res.body.articles.forEach((article) => {
-                expect(typeof article.title).toBe('string');
-                expect(typeof article.topic).toBe('string');
-                expect(typeof article.author).toBe('string');
-                expect(typeof article.body).toBe('undefined'); //Check body has been removed
-                expect(typeof article.created_at).toBe('string');
-                expect(typeof article.article_img_url).toBe('string');
-                expect(typeof article.comment_count).toBe('string');
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((res) => {
+                //Check Correct Length
+                expect(res.body.articles.length).toBeGreaterThanOrEqual(13);
+                //Check Sort Order
+                expect(res.body.articles).toBeSorted('created_at', 'desc');
+                expect(res.body.articles[12].title).toBe("Z");
+                //Check each element has correct props
+                res.body.articles.forEach((article) => {
+                    expect(typeof article.title).toBe('string');
+                    expect(typeof article.topic).toBe('string');
+                    expect(typeof article.author).toBe('string');
+                    expect(typeof article.body).toBe('undefined'); //Check body has been removed
+                    expect(typeof article.created_at).toBe('string');
+                    expect(typeof article.article_img_url).toBe('string');
+                    expect(typeof article.comment_count).toBe('string');
+                });
+            });
+        });
+        test('/api/articles?topic=valid-topic 200: returns articles filtered by a valid topic', () => {
+            return request(app)
+            .get('/api/articles?topic=mitch')
+            .expect(200)
+            .then((res) => {
+                //Check Correct Length
+                expect(res.body.articles.length).toBeGreaterThanOrEqual(12);
+                //Check Sort Order
+                expect(res.body.articles).toBeSorted('created_at', 'desc');

```

Comment:
i would put this in a separate test as this is a different behaviour to what is being described in the test description - testing that the articles filtered by topic are still sorted, rather than just checking they are filtered